src/components/NcModal/NcModal.vue

```diff
@@ -1019,7 +1019,7 @@ export default {
 			z-index: 1;
 			position: absolute;
 			top: 4px;
-			right: 4px;
+			inset-inline-end: 4px;
```

Comment:
Just because this place is touched
```suggestion
			inset-inline-end: var(--default-grid-baseline);
```